pkg/scheduler/framework/runtime/framework.go

```diff
@@ -648,9 +645,8 @@ func (f *frameworkImpl) RunScorePlugins(ctx context.Context, state *framework.Cy
 		}
```

Comment:
Let's make the Errorf construction above consistent with `runScorePlugin`. Use `%w`, like in #94695.